tools/star_rod_enum_to_decomp.py

```diff
@@ -34,11 +27,11 @@ def create_enum(file_content, prefix, ordering):
 
     for (key, value) in re.findall(r'(\S+)\s+=\s+(\S+)', file_content):
         if ordering:
-            key = '_'.join(re.sub(r'([A-Z])', r' \1', key).split()).replace("H_P", "HP").replace("F_P", "FP").replace("J_P", "JP").replace("N_P_C", "NPC").replace("__", "_").replace("-", "")
+            key = '_'.join(re.sub(r'([A-Z]{1,2})', r' \1', key).split()).replace("N_P_C", "NPC").replace("__", "_").replace("-", "")
             key = prefix.upper() + '_{:<{width}}'.format(key, width=max_size + 2).upper()
             ret += "    " + key + " = 0x" + '{:>{fill}{width}}'.format(value, fill=0, width=8) + ",\n"
         else:
-            value = '_'.join(re.sub(r'([A-Z])', r' \1', value).split()).replace("H_P", "HP").replace("F_P", "FP").replace("J_P", "JP").replace("N_P_C", "NPC").replace("__", "_").replace("-", "")
```

Comment:
What happened to `.replace("H_P", "HP").replace("F_P", "FP").replace("J_P", "JP")`

Reply:
No longer needed as the regex can detect those properly now